contribs/gmf/apps/desktop/Controller.js

```diff
@@ -154,6 +154,8 @@ class Controller extends AbstractDesktopController {
  */
 const module = angular.module('Appdesktop', [appBase.name, gmfControllersAbstractDesktopController.name]);
 
+module.value('gmfDropFileEnabled', true);
```

Comment:
I think we don't need this value anymore. You can remove it. (also the other occurrences to this value)

Or a (better ?) alternative would be to use this value and remove what we have done to disable the drag&drop overlay on mobile (but keep this https://github.com/camptocamp/ngeo/pull/5989/files#diff-84492b8aaea37a9f09e40541ee5693b3R45 and set the `if` to use this `gmfDropFileEnabled` file)